packages/core/src/app/shipping/MultiShippingFormV2.test.tsx

```diff
@@ -189,6 +189,8 @@ describe('MultiShippingFormV2 Component', () => {
 
         expect(allocateItemsModalHeader).toBeInTheDocument();
 
+        expect(screen.getByText('Shipping isn\'t required for digital products, thus it\'s not displayed here.')).toBeInTheDocument();
```

Comment:
The above mock checkout data doesn't have digital items in the cart, curious to know that how are we able to assert this message?

Reply:
https://github.com/bigcommerce/checkout-js/blob/5f25ba2e45ec698bddd842b639f8a8b0b6cd92fa/packages/core/src/app/cart/carts.mock.ts#L25

The mock cart has digital item.

Reply:
I updated the existing mock and created a new test